### PR TITLE
feat: compose paragraph

### PR DESCRIPTION
**Pull request by Barsnes**
I did every component in singular commits so it's easier to check.

Did not do anything with form components.

Excluding `Card` since it was only on content and footer, which is deleted in another PR

### CODE REVIEW

File: packages/css/tabs.css [hunk 1]
```diff
@@ -6,14 +6,38 @@
   --dsc-tabs-content-color: var(--ds-color-neutral-text-default);
   --dsc-tabs-list-border-color: var(--ds-color-neutral-border-subtle);
 
+  & .ds-tabs__tab {
+    @composes ds-body-text--short-md from './baseline/typography.css';
+  }
+
+  & .ds-tabs__panel {
+    @composes ds-body-text--short-md from './baseline/typography.css';
```
Comment (eirikbacker) [inline packages/css/tabs.css:14]: Skulle disse heller vært satt på `ds-tabs` i stede? ☺️

File: packages/react/src/components/Accordion/AccordionHeading.tsx [hunk 2]
```diff
@@ -15,18 +15,7 @@ export type AccordionHeadingProps = {
...
-            'ds-focus ds-paragraph ds-paragraph--sm ds-line-height--md', // Very TMP fix awaiting CSS modules
-            className,
-          )}
-          ref={ref}
-          {...rest}
-        />
-      </Paragraph>
-    );
+  function AccordionHeading(rest, ref) {
+    return <u-summary ref={ref} {...rest} />;
```
Comment (eirikbacker) [inline packages/react/src/components/Accordion/AccordionHeading.tsx:19]:
```suggestion
  function AccordionHeading({ className, ...rest }, ref) {
    return <u-summary class={className} ref={ref} {...rest} />;
```
...for å fikse React sin custom element foreståelse :)

Comment (Barsnes) [reply packages/react/src/components/Accordion/AccordionHeading.tsx:19]: Den er god, legger også inn kommentar om det, for framtiden

**Review by eirikbacker — APPROVED** [review]
Nydelig jobba 👏

**Pull request merged**